medical_pharmacy_us/models/medical_pharmacy.py

```diff
@@ -17,7 +17,7 @@ class MedicalPharmacy(models.Model):
         inverse=lambda s: s._inverse_identification(
             'napb_num', 'NAPB',
         ),
-        help='National Boards of Pharmacy Id #',
+        help='National Boards of Pharmacy Id # - seven-digit number',
```

Comment:
Switching between hyphenated and non with `number-digit` vs `number digits`. Again indifferent, but keep it consistent.